### ISSUE TITLE
4.1.0: Unable to resolve "./util.inspect" in `swell.umd.min.js`

### ISSUE THREAD

**Issue by lukifer** [swellstores/swell-js#176]
When trying to update from 4.0.0 to 4.1.0, I'm still encountering [this issue](https://github.com/swellstores/swell-js/issues/172) with `util.inspect`:
```
ERROR in ../node_modules/.pnpm/swell-js@4.1.0/node_modules/swell-js/dist/swell.umd.min.js
Module not found: Error: Can't resolve './util.inspect' in '[...]node_modules/.pnpm/swell-js@4.1.0/node_modules/swell-js/dist'
 @ ../node_modules/.pnpm/swell-js@4.1.0/node_modules/swell-js/dist/swell.umd.min.js 752:20-45
 @ ./dist/swell.plugin.4760da49.js
 @ ./dist/index.js
 @ ./dist/client.js
 @ multi ./dist/client.js
```

**Comment by awwit** [comment]
@lukifer thank you for report! This problem was fixed in version `4.1.1`. We had some glitches when publishing version `4.1.0` due to which the latest changes were not included. Everything should be fine now!

**Comment by lukifer** [comment]
Seems to be resolved now, thanks!
